tests/serve/test_vllm.py

```diff
@@ -34,6 +36,22 @@ class VLLMConfig(EngineConfig):
     WORKSPACE_DIR, "examples/backends/vllm"
 )
 
+# Load data for multimodal b64 passthrough, fallback to stub if not available
+B64_IMG = None
+B64_EXPECTED_RESPONSE = []
+
+try:
+    with open(MULTIMODAL_IMG_PATH, "rb") as f:
+        B64_IMG = base64.b64encode(f.read()).decode()
+        B64_EXPECTED_RESPONSE = ["purple"]
+except FileNotFoundError:
+    logger.info(
+        f"Multimodal asset not found at {MULTIMODAL_IMG_PATH}, using 1x1 PNG stub for basic validation"
+    )
+    # 1x1 transparent PNG fallback stub
+    B64_IMG = "iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAAAAAA6fptVAAAACklEQVR4nGNoAAAAggCBd81ytgAAAABJRU5ErkJggg=="
+    B64_EXPECTED_RESPONSE = []
```

Comment:
Why fallback? Don't we want to fail if our expected test input is not available? I'm concerned this will hide someone changing/moving the path and then we lose the test coverage on a real image without noticing

Reply:
> My opinion is that the logger.info(...) in logs would be enough to catch any failures in the vLLM job.

If the test passes, I think most people generally won't open or read the logs - I think it will be a bit of a needle in a haystack to notice later on

Reply:
Is there no simpler way to avoid this test_vllm.py code getting hit at all for sglang/trtllm? Maybe can pick someone else's brain as well